object-assign/object-assign.d.ts

```diff
@@ -4,6 +4,5 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare module "object-assign" {
-  function objectAssign(target: any, ...sources: any[]): any;
-  export = objectAssign;
+  export default function(target: any, ...sources: any[]): any;
```

Comment:
`import zip = require("./ZipCodeValidator");` is not same as `import zip from "./ZipCodeValidator";`.
`import zip = require("./ZipCodeValidator");` is CommonJS style.